zeppelin/hdi_add_zeppelin.sh

```diff
@@ -0,0 +1,220 @@
+#! /bin/bash
+
+#----------------------------------------------------------------
+# THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
+#----------------------------------------------------------------
+
+
+#----------------------------------------------------------------
+# PRINT USAGE INFORMATION
+#----------------------------------------------------------------
+
+
+print_usage()
```

Comment:
what are the cluster versions that we are going to support? Is there a way to fail fast in case the cluster version is not supported?

Reply:
we will support only 2.6